File: M3-Programacion/UF2/Practica3-M03/Exercici1.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Exercici1 {
	
	public static int sumar(int num1,int num2) {
		return num1+num2;
	}
	
	public static int restar(int num1,int num2) {
		return num1-num2;
	}
	
	public static int div(int num1,int num2) {
		return num1/num2;
	}
	
	public static int mult(int num1,int num2) {
		return num1*num2;
	}
	
	public static void main(String[] args) throws IOException {
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		String continuar ="s";
		do {
			System.out.print("Pon un número");
			int num1 = Integer.parseInt(reader.readLine());
			System.out.print("Pon un operador (+ - * /)");
			String op = reader.readLine();
			System.out.print("Pon un número");
			int num2 = Integer.parseInt(reader.readLine());
			
			switch (op) {
			case "x":
				System.out.println(mult(num1, num2));
				break;
			case "*":
				System.out.println(mult(num1, num2));
				break;
			case "/":
				if (num2 == 0) {
					System.out.println("Error división entre zero");
				}else {
					System.out.println(div(num1, num2));
				}
				break;
			case "+":
				System.out.println(sumar(num1, num2));
				break;
			case "-":
				System.out.println(restar(num1, num2));
				break;

			default:
				System.out.println("Error al introducir los datos");
				break;
			}
			System.out.println("Quieres continuar s/n ? ");
			continuar = reader.readLine();
			if (continuar.equals("n")) {
				break;
			}
		} while (continuar.equals("s"));
	}
} 

File: M3-Programacion/UF2/Practica3-M03/Exercici1.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

def sumar(num1, num2):
	return num1+num2

def restar(num1, num2):
	return num1-num2

def div(num1, num2):
	return num1/num2

def mult(num1, num2):
	return num1*num2

condition = True
while condition:
	num1 = int(input("Pon un número: "))
	op = input("Pon un operador (+ - * /)")
	num2 = int(input("Pon un número: "))
	
	if (op=="x"):
		print(mult(num1,num2))
	
	elif (op=="*"):
		print(mult(num1,num2))
	
	elif (op=="-"):
		print(restar(num1,num2))
	
	elif (op=="/"):
		if (num2==0):
			print("Error división entre zero")
		else:
			print(div(num1,num2))
	
	elif (op=="+"):
		print(sumar(num1,num2))
	
	else:
		print("Error al introducir los datos")
		
	condition2 = input("Quieres continuar s/n ? ")
	if (condition2 == "n"):
		condition = False
